aospy/data_loader.py

```diff
@@ -296,7 +296,7 @@ def recursively_compute_variable(self, var, start_date=None, end_date=None,
             data = [self.recursively_compute_variable(
                 v, start_date, end_date, time_offset, **DataAttrs)
                     for v in var.variables]
-            return var.func(*data)
+            return var.func(*data).rename(var.name)
```

Comment:
Indeed, good suggestion to add a test for this.  One minor change and things work smoothly!